src/App.tsx

```diff
@@ -67,18 +53,14 @@ export const App: React.FC = () => {
                   </div>
                 )}
 
-                {author && loaded && !hasError && posts.length === 0 && (
+                {author && loading && !err && posts.length === 0 && (
                   <div className="notification is-warning" data-cy="NoPostsYet">
                     No posts yet
                   </div>
                 )}
```

Comment:
The condition for displaying the 'No posts yet' message should be when `loading` is false, not true. Consider changing the condition to `{author && !loading && !err && posts.length === 0 && ( ... )}`.